entry_tasks/13_coordinate/main.py

```diff
@@ -0,0 +1,6 @@
+if __name__ == "__main__":
+    a = int(input('>>> '))
+    b = int(input('>>> '))
+    c = int(input('>>> '))
+    d = tuple([a, b, c])
```

Comment:
How about to not create extra list in memory:
`d = (a, b, c)`

Reply:
@mykytapavlov Done.